english/bored/work/work.txt

```diff
@@ -0,0 +1 @@
+STOP MESSING AROUND AND GET BACK TO WORK! 
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.